lib/spack/spack/test/bootstrap.py

```diff
@@ -150,3 +150,20 @@ def test_nested_use_of_context_manager(mutable_config):
         with spack.bootstrap.ensure_bootstrap_configuration():
             assert spack.config.config != user_config
     assert spack.config.config == user_config
+
+
+@pytest.mark.parametrize('path_ev,expected_missing', [
+    (None, False),
```

Comment:
I think this succeeds because the system where the test is being performed happens to include all the bootstrap files. IMO to make this test more independent, I would recommend creating an explicit separate path with `tmpdir`, then adding "noop" executables with the expected names.

Reply:
See f7a4c86